dask/blockwise.py

```diff
@@ -367,7 +367,7 @@ def make_blockwise_graph(func, output, out_indices, *arrind_pairs, **kwargs):
                 all_indices.add(x)
     assert set(numblocks) == block_names
 
-    dummy_indices = all_indices - set(out_indices)
+    dummy_indices = sorted(all_indices - set(out_indices))
```

Comment:
This is to make the internal deterministic, and was helpful for debugging, as later iteration/enumeration were changing order on each re-run.

Reply:
Just to be conservative, can you revert this? I don't know what the performance costs of this are for large graphs, and I don't trust our benchmarks to pick them up.